src/core/mod.rs

```diff
@@ -292,16 +290,38 @@ pub const STACKS_EPOCHS_MAINNET: &[StacksEpoch] = &[
         epoch_id: StacksEpochId::Epoch10,
         start_height: 0,
         end_height: BITCOIN_MAINNET_FIRST_BLOCK_HEIGHT,
+        block_limit: ExecutionCost {
```

Comment:
Stacks 1.0 didn't have smart contracts, so the ExecutionCost for it should be zero in all dimensions.

Reply:
Updated as of 340985579